gulpfile.js

```diff
@@ -82,6 +83,58 @@ function createReloadServer() {
   });
 }
 
+function writeServiceWorkerFile() {
+  var path = require('path');
+  var rootDir = 'dist';
+
+  var config = {
+    cacheId: 'polymerproject',
+    staticFileGlobs: [
+      rootDir + '/1.0/index.html',
```

Comment:
If that's the case, then a single entry in `dynamicUrlToDependencies` does seem required:

```js
dynamicUrlToDependencies: {
  'dist/1.0/': ['path/to/templates/head-meta.html', 'path/to/dist/1.0/index.html']
}
```

(This will take care of both requests for 'dist/1.0/' and 'dist/1.0/index.html' due to [`directoytIndex`](https://github.com/GoogleChrome/sw-precache#directoryindex-string)'s default behavior.